pkg/encryption/cipher.go

```diff
@@ -113,16 +110,39 @@ func (c *Cipher) Decrypt(s string) (string, error) {
 		return "", fmt.Errorf("failed to decrypt cookie value %s", err)
 	}
 
-	if len(encrypted) < aes.BlockSize {
-		return "", fmt.Errorf("encrypted cookie value should be "+
+	decrypted, err := c.DecryptCFB(encrypted)
+	if err != nil {
+		return "", err
+	}
+
+	return string(decrypted), nil
+}
+
+// Encrypt with AES CFB on raw bytes
+func (c *Cipher) EncryptCFB(value []byte) ([]byte, error) {
```

Comment:
These are split out now from the legacy `Encrypt` and `Decrypt` to not be forced to do a base64 encoding in all cases.

This would be where we add `EncryptGCM` support if desired to speed up the encryption/decryption layer.